### ISSUE TITLE
Separate DB Driver in Config

### ISSUE THREAD

**Issue by hadihammurabi** [gowok/gowok#1]
before:
```yaml
...
databases:
  default:
    driver: postgresql
    dsn: 'host=localhost user=postgres password= database=postgres port=5432 sslmode=disable'
  secondary:
    driver: mongo
    dsn: mongodb://root:root@localhost
  cache:
    driver: redis
...
```

after:
```yaml
...
sql:
  default:
    driver: postgresql
    dsn: 'host=localhost user=postgres password= database=postgres port=5432 sslmode=disable'
    enabled: true

mongodb:
  default:
    driver: mongodb
    dsn: mongodb://root:root@localhost
    enabled: true

cache:
  default:
    driver: redis
    enabled: true
...
```

It makes DB usage in application easier and cleaner.

**Comment by hadihammurabi** [comment]
in v0.3.2 the config is like that

```yaml
...
sql:
  default:
    driver: postgresql
    dsn: 'host=localhost user=postgres password= database=postgres port=5432 sslmode=disable'

mongodb:
  default:
    dsn: mongodb://root:root@localhost

cache:
  default:
    driver: redis
...
```

**Comment by hadihammurabi** [comment]
resolved at #9a79d03
